labgrid/driver/serialdriver.py

```diff
@@ -22,11 +22,11 @@ class SerialDriver(ConsoleExpectMixin, Driver, ConsoleProtocol):
     """
     # pyserial 3.2.1 does not support RFC2217 under Python 3
     # https://github.com/pyserial/pyserial/pull/183
-    if tuple(int(x) for x in serial.__version__.split('.')) <= (3, 2, 1):
+    if tuple(x for x in serial.__version__.split('.')) <= ('3', '2', '1'):
```

Comment:
We should switch this over to use `version` from the `packaging` module. We require setuptools to run setup.py anyway and it includes the packaging module.
```suggestion
    if version(serial.__version__) <= version('3.2.1'):
```

Reply:
setuptools is a buildtime dependency but this makes packaging a runtime dependency, so I specified it as one.

Reply:
Yes, I agree its better to be explicit here. Considering that we advocate for an installation from git we require setuptools anyway, but I hope to eventually make the pypi installation route for pytest usage available again.